### ISSUE TITLE
Remove unecessary/redundant concrete testing classes for each abstract set

### ISSUE THREAD

**Issue by jboreiko** [palantir/atlasdb#392]
Each KVS should be tested against a single suite of tests.  Having a suite that is extended for each KVS will reduce the number of floating concrete test classes in the code base.  An example of a set of redundant KVS concrete test classes is 
- https://github.com/palantir/atlasdb/blob/f552573609dba1290e4068d710475548df4d7bb1/atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceSweeperTest.java
- https://github.com/palantir/atlasdb/blob/f552573609dba1290e4068d710475548df4d7bb1/atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceTransactionTest.java
- https://github.com/palantir/atlasdb/blob/f552573609dba1290e4068d710475548df4d7bb1/atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceSerializableTransactionTest.java

This pattern is repeated across all KVS's and makes it difficult to determine which tests are necessary for validating a KVS to be fully functional with all major features.

@rjullman

**Comment by jboreiko** [comment]
Currently blocked waiting on DbKVS move.

**Comment by gsheasby** [comment]
Should verify that Cassandra, Postgres and Oracle KVSs do not have obvious holes in their testing, and then close this.

**Comment by rhero** [comment]
@jboreiko has this issue been covered? Or is there a specific open action still?

**Comment by jboreiko** [comment]
Nope, not resolved, but like much of the handoff checklist work it is unguided refactoring that we have chosen to avoid until necessary.  Would be happy to close as won't fix for the time being.

**Comment by gmaretic** [comment]
Given that we opted, at least for Cassandra, to have standalone test classes that can be run in isolation instead of a giant suite, this won't be fixed
